### PR TITLE
fix: remove logging while collecting GPU metrics

### PR DESCRIPTION
**Pull request by vprashar2929**
removes debugging logging from GPU metrics

### CODE REVIEW

**Comment by vprashar2929** [comment]
cc: @dave-tucker

**Review by dave-tucker — APPROVED** [review]
LGTM. The line can also be removed entirely - it was left over from debugging and I'd forgotten to remove it.

**Comment by vprashar2929** [comment]
Ah in that case I am happy to remove it as well

**Comment by vprashar2929** [comment]
cc: @rootfs

File: pkg/metrics/utils/utils.go [hunk 1]
```diff
@@ -57,7 +57,6 @@ func CollectResUtilizationMetrics(ch chan<- prometheus.Metric, instance interfac
 			CollectResUtil(ch, instance, collectorName, collectors[collectorName])
 		}
 	}
-	klog.Info("Collecting GPU metrics")
```
Comment (sthaha) [inline pkg/metrics/utils/utils.go:60]: We may want to log this somewhere at V(5) or so  probably inside the `if`

**Review by sthaha — APPROVED** [review]

**Pull request merged**